src/style.css

```diff
@@ -1 +1,80 @@
+/* stylelint-disable string-quotes */
+/* stylelint-disable color-hex-case */
+/* stylelint-disable no-missing-end-of-source-newline */
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
```

Comment:
do not forget to add more fallback fonts
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/cff5cecd-7624-4602-8a08-b09639863e8d)